File: Python/Task06/n38940.py
# https://inf.reshuege.ru/problem?id=38940

count = 0
for i in range(1000000):
	s = i
	s = 3 * (s // 10)
	n = 1
	while s < 221:
		s = s + 13
		n = n * 2
	if n == 128:
		count += 1
print(count)

File: C++/Task06/n38940/n38940.cpp
// https://inf-ege.sdamgia.ru/problem?id=38940

#include <iostream>

int main()
{
	int count = 0;
	for (int i = 0; i < 1000000; i++)
	{
		int s, n;
		s = i;
		s = 3 * (s / 10);
		n = 1;
		while (s < 221) {
			s = s + 13;
			n = n * 2;
		}
		if (n == 128)
		{
			count += 1;
		}
	}
	std::cout << count << std::endl;
	return 0;
}